tests/codegen/riscv-abi/riscv64-lp64d-abi.rs

```diff
@@ -4,7 +4,7 @@
 //@ only-linux
 #![crate_type = "lib"]
```

Comment:
If you convert these codegen tests to `no_core`, they'll run on any host/target (even without the riscv target built), and as a result they'll be automatically tested in CI.

Generally for simple codegen tests you just need to stub out some lang items and explicitly specify the target--see 8fdd5e044b6bcbb42b4342a19a62ca501fe33b58 for an example where I converted one of the loongarch tests.

Reply:
Thanks for the example! That really helped!